.changeset/stupid-pens-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/core-app-api': patch
```

Comment:
We are not able to remove the `homepage` right away as the package is 1.x but let's mark it as deprecated for now in the config.d.ts. We are going to release a new major version of core-app-api in a couple of months then we can remove this as well as it's deprecated then.

Move it to a separate changeset as a `minor` change

Reply:
Ok, sounds good. Will work on this later this week

Reply:
Yes almost, I think we should delete it from app-config.yaml as well :)

Reply:
Ok, it's gone now 👍